content/en/platform/corda/5.0-beta/developing/development-tutorials/build-basic-cordapp/basic-cordapp-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+date: '2023-01-11'
+title: "Initial setup"
+menu:
+  corda-5-beta:
+    parent: corda-5-beta-tutorial-develop-building-your-first-basic-cordapp
+    identifier: corda-5-beta-tutorial-develop-initial-setup
+    weight: 1100
+section_menu: corda-5-beta
+---
+
```

Comment:
@lankydan Could you please add a short intro and learnin objectives? Similar to what we have in other tutorials - what the user is going to do and what are the learning objectives? For example, where do I create those module and what are they exactly? Why do I need them? Why do I need to create packages and what are they? We want to make sure that somebody who's just started learning Corda knows exactly what is that they're doing and why they're doing it.

Reply:
Create these modules in the project's root directory. These modules contain the required setup to build your CorDapp. The `ledger-utxo-example-apples-app` is where you will write your application logic and `ledger-utxo-example-apples-contract` is where your object definitions will be placed.

The packages are not "required" but allows us to define a suggested structure that users follow.

Reply:
Updated.